adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -451,7 +451,7 @@ private AuthenticationConstants() { }
          * Signature info for Azure authenticator app that installs authenticator
          * component.
          */
-        public static final String AZURE_AUTHENTICATOR_APP_SIGNATURE = "ho040S3ffZkmxqtQrSwpTVOn9r0=";
```

Comment:
In order to automate webviews in an Android APK, a setting to allow remote webview debugging has to be turned on. My assumption is that building with that setting turned on changed the signature.